src/stories/flows/eligibility/data/pages.flow.tsx

```diff
@@ -16,26 +16,25 @@ export const pages: IPages = {
     type:    PAGE_TYPE.LANDING,
   },
   noResultsPage: {
+    children: (<>You can <a href="#apply">apply for benefits</a> but based on what you told us today, you may not be eligible at this time.</>),
     footer:
       'The information you shared today tells us about your current situation. In the future, you may become eligible for benefits as you age or things in your life change.',
-    id:       PAGE_TYPE.NO_RESULTS,
-    section:  { id: PAGE_TYPE.RESULTS },
-    subTitle:
-      'You can <a href="#">apply for benefits</a> but based on what you told us today, you may not be eligible at this time.',
-    title: 'You may not be eligible for benefits.',
-    type:  PAGE_TYPE.NO_RESULTS,
+    id:      PAGE_TYPE.NO_RESULTS,
+    section: { id: PAGE_TYPE.RESULTS },
+    title:   'You may not be eligible for benefits.',
+    type:    PAGE_TYPE.NO_RESULTS,
   },
   resultsPage: {
-    body: `This gives you an idea of what you may be eligible for, but we can't guarantee you'll get these benefits. We'll analyze more details during the application process to provide a definite answer.
-<p/>
-The information you shared today tells us about your current situation. In the future, you may become eligible for other benefits as you age or things in your life change.`,
     bodyHeader:    'Benefits you may be eligible for',
     bodySubHeader: "Here's what you may be eligible for and why",
-    id:            PAGE_TYPE.RESULTS,
-    info:          'Each benefit we provide has an official name. You may see these names in other materials or hear our employees use them on the phone and in our offices.',
-    section:       { id: PAGE_TYPE.RESULTS },
-    title:         'You may be eligible for benefits.',
-    type:          PAGE_TYPE.RESULTS,
+    children:      (<>This gives you an idea of what you may be eligible for, but we can{"'"}t guarantee you{"'"}ll get these benefits. We{"'"}ll analyze more details during the application process to provide a definite answer.
```

Comment:
`{"'"}` is really ugly. Also, it will also be tricky for content editors once it gets moved into drupal and content editors try to work with it.
Maybe put the whole sentence into `{"` like:
`{"This gives you an idea of what you may be eligible for, but we can't guarantee you'll get these benefits. We'll analyze more details during the application process to provide a definite answer."}`?
Not a big deal, just an interesting formatting/escaping challenge.